guide/app_star_catalogue.tex

```diff
@@ -243,14 +256,15 @@ \subsubsection{Star Data Records}%\label{star-data-records}
 \begin{table}[htbp]
 \begin{tabularx}{\textwidth}{llllX}\toprule
 \emph{Name} & \emph{Offset}& \emph{Type} & \emph{Size} &\emph{Description}\\\midrule
-x0   &  0      & int          & 20 bits & position of the star relative to the central point in the star's zone, in axis 1\\%\midrule
-x1   & 20 bits & int          & 20 bits & position of the star relative to the central point in the star's zone, in axis 2\\%\midrule
-dx0  & 40 bits & int          & 14 bits & proper motion of the star in axis 1\\%\midrule
-dx1  & 54 bits & int          & 14 bits & proper motion of the star in axis 2\\%\midrule
-b\_v & 68 bits & unsigned int &  7 bits & magnitude level in B-V colour. This value refers to one of 128 discrete steps 
-                                          in the magnitude range for the file\\%\midrule
-mag  & 75 bits & unsigned int &  5 bits & magnitude level in the V-I colour. This value refers to one of 32 discrete steps 
-                                          in the magnitude range for the file\\\bottomrule
+gaia_id        &  0 & long long          & 8 & Gaia source id\\%\midrule
+x0             &  8 & int                & 4 & Right ascension $\alpha$ of the star in mas unit.\\%\midrule
+x1             & 12 & int                & 4 & Declination $\delta$ of the star in mas unit.\\%\midrule
+dx0            & 16 & int                & 4 & Proper motion $\mu_{\alpha}$ of the star in $mu$as/year%\midrule
```

Comment:
This and next 3 lines: again, please use \\\\ as line end marker (The "%\midrule" is a testwise commented-away previous horizontal line and can go away. )

Also, this and next line, use "$\mu$as/year"  (with backslash) to typeset "&mu;as/year"